source/OptimaJet.DWKit.StarterApplication/package-lock.json

```diff
@@ -5,24 +5,9 @@
   "requires": true,
   "dependencies": {
     "@aspnet/signalr": {
-      "version": "1.1.0",

```

Comment:
I am concerned about version of SignalR.  It was 1.1.0 and now it looks like it resolves to 1.0.4.